lib/common/test_utils.go

```diff
@@ -132,6 +133,11 @@ func DropTables(db *sql.DB) {
 		log.Fatalf("an error '%s' was not expected when dropping the index", err)
 	}
 
+	_, err = tx.Exec("DROP INDEX IF EXISTS sentMessageTotal_unique")
```

Comment:
I wonder if it's possible later to create   `0_` prefixed migration that drops all the tables that we create to avoid having to remember to add these here DROPs here :thinking: